platform/kobo/enable-wifi.sh

```diff
@@ -3,7 +3,8 @@
 # Load wifi modules and enable wifi.
 
 lsmod | grep -q sdio_wifi_pwr || insmod /drivers/$PLATFORM/wifi/sdio_wifi_pwr.ko
-lsmod | grep -q dhd || insmod /drivers/$PLATFORM/wifi/dhd.ko
+# WIFI_MODULE_PATH = /drivers/$PLATFORM/wifi/$WIFI_MODULE.ko
+lsmod | grep -q $WIFI_MODULE || insmod $WIFI_MODULE_PATH
```

Comment:
Well yes, but I don't think that necessarily means it was already set, say, five years ago. To partially answer my own question, this is in the /etc/init.d/rcS from when I first got my H2O back in 2014:

```bash
INTERFACE=wlan0
WIFI_MODULE=ar6000
if [ $PLATFORM != freescale ]; then
	INTERFACE=eth0
	WIFI_MODULE=dhd
fi

export PLATFORM
export PRODUCT
export INTERFACE
export WIFI_MODULE

export NICKEL_HOME=/mnt/onboard/.kobo
export LD_LIBRARY_PATH=/usr/local/Kobo
export WIFI_MODULE_PATH=/drivers/$PLATFORM/wifi/$WIFI_MODULE.ko
export LANG=en_US.UTF-8
```

A quick search shows that the relevant variable was already there in 2012: https://github.com/matsuu/KoboRoot/blob/1c648a040c3afe1899a22e8e2d4357d395e5b46c/etc/init.d/rcS#L97-L102

So I can stop worrying. ;-)